source/sass/components/primary-nav.scss

```diff
@@ -268,7 +268,7 @@
           font-weight: 500;
           transition: transform 0.15s;
           display: inline-block;
-          margin: 10px 0;
+          margin: 1.25rem 0;
```

Comment:
I thought about using a Tailwind class for this instead. However, there are multiple places outside of PNI affected by the stylings we define here. If we want to replace this line with Tailwind class, it means we have to update multiple template files to include the Tailwind class and that seems out of the scope for this ticket.